solr/test-framework/src/java/org/apache/solr/SolrTestCase.java

```diff
@@ -225,7 +238,7 @@ public static void setDefaultConfigDirSysPropIfNotSet() throws Exception {
     System.setProperty("solr.clustering.enabled", "false");
     System.setProperty("solr.peerSync.useRangeVersions", String.valueOf(random().nextBoolean()));
     System.setProperty("zookeeper.nio.directBufferBytes", Integer.toString(32 * 1024 * 2));
-    System.setProperty("solr.disablePublicKeyHandler", "true");
+    enableReuseOfCryptoKeys();
```

Comment:
Looks like I had this reversed. We should likely enable reuse below in the non Nightly run and disable for the closer to prod Nightly run.